gatsby/pages.rst

```diff
@@ -1,7 +1,8 @@
 Pages
 =====
 
-Pages are the basic components of a GatsbyJS site.
+Pages can be considered similar to HTML documents in a basic website as they behave that way, but here, internally
+they are React components instead with way more functionality and can even be dynamically generated.
 
```

Comment:
I would remove the word basic here, what is basic, from when on something is not basic anymore ?

Reply:
Done! :)

Reply:
I made the change